scenario_id: test_broken_scenario
name: Тестовый сценарий с ошибками
description: Демонстрация автоисправления валидатором
steps:
- id: start
  type: start
  params:
    message: Начинаем тест
  next_step: send_msg
- id: send_msg
  type: channel_action
  params:
    text: Привет!
    action: send_message
    chat_id: '{chat_id}'
  next_step: get_user_input
- id: get_user_input
  type: input
  params:
    prompt: Введите что-нибудь
  next_step: save_response
- id: save_response
  type: mongo_upsert_document
  params:
    data: '{user_input}'
    collection: default_collection
  next_step: check_condition
- id: check_condition
  type: branch
  params:
    condition: '{user_input} != '''''
    true_step: create_contact
    false_step: end
- id: create_contact
  type: amocrm_create_contact
  params:
    name: '{user_input}'
    phone: '+1234567890'
  next_step: ai_response
- id: ai_response
  type: llm_query
  params:
    prompt: 'Ответь пользователю: {user_input}'
  next_step: send_result
- id: send_result
  type: channel_action
  params:
    text: 'Результат: {ai_response}'
    buttons:
    - text: Готово
      callback_data: done
    action: send_message
    chat_id: '{chat_id}'
  next_step: end
- id: end
  type: end
  params:
    message: Конец
